app/Http/Controllers/BuyersController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Buyer;
+use App\Models\Address;
+use Illuminate\Http\Request;
+
+class BuyersController extends Controller
+{
+    public function showOne($id)
```

Comment:
Не указан тип входного параметра
```suggestion
    public function showOne(int $id)
```